### PR TITLE
Map Popup Adjustment for Smaller Laptop Screens

### PR DESCRIPTION
**Pull request by solderq35**
Issue Referenced: https://github.com/OSU-Sustainability-Office/energy-dashboard/issues/190

Testing procedure:
1. Start webpage locally.

2. While browser is in fullscreen mode, enter browser, Inspect Element > Console, then type `console.log(window.innerHeight)`

    - For me I have 722 for `window.innerHeight` value. If your value is different, you may have to adjust the `Vue.prototype.$WINDOW_HEIGHT` value in `src/main.js` to be higher or lower before proceeding. 
    - For testing purposes, the `$WINDOW_HEIGHT` value should be slightly lower than what your actual `window.innerHeight` value is (I set `$WINDOW_HEIGHT` to 700 which is a bit below 722).

3. Return to browser in fullscreen mode, click on a building and observe popup.

4. Close the popup. 

5. Then set browser to windowed mode with a `window.innerHeight` value lower than your `$WINDOW_HEIGHT` value (check `window.innerHeight` in dev console to be sure), click on a building and observe popup.

Notes:
- After you adjust the browser window size, you need to close the current popup and re-open it for the "map popup mode" to adjust.
- Reloading the webpage is not necessary
- Hard-coded values I'm not sure about I've commented about in the code.
- `$WINDOW_HEIGHT` will need to be adjusted (see edit below).

Screenshots:
full screen (`window.innerHeight` = 722)
full screen

windowed (`window.innerHeight` = 612)
windowed

EDIT 12/24/2022:
After testing with a page ruler chrome extension (https://chrome.google.com/webstore/detail/page-ruler/jcbmcnpepaddcedmjdcmhbekjhbfnlff), I have concluded that we need a `$WINDOW_HEIGHT` value of at least 866, 900 to be safe, in order for the "tall mode" of the map popup to fully show up without need of scrolling. Window inner height measures from top of website navbar to the windows navbar below (or whatever the equivalent on other OS).

The `$WINDOW_HEIGHT` value of 700 currently is just for testing, will need to be adjusted.

- Note: Testing with ruler above was done by clicking on the map popup, then zooming out. If you don't click on the popup first, the popup will not "zoom out with you". 
- Zooming out makes the ruler detect more pixel length (and seems accurate), but it doesn't change the value you get when typing `console.log(window.innerHeight)` into browser dev console. Due to this I (or someone else) will have to test this sometime on a bigger monitor (mine only supports up to 1920 x 1080 screen resolution).

Reference Image:
Reference Image

### CODE REVIEW

**Comment by solderq35** [comment]
merged master into feature branch to do test build with new github actions pipeline

File: src/components/map/sideView.vue [hunk 1]
```diff
@@ -2,50 +2,95 @@
...
     </el-row>
-  </el-row>
+  </div>
 </template>
+
 <script>
 import chartController from '@/components/charts/chartController'
 import switchButtons from '@/components/map/time_switch_buttons_big'
 
+console.log('Inner Height: ' + window.innerHeight)  // or you can just call this in the dev browser console
```
Comment (MilanDonhowe) [inline src/components/map/sideView.vue:92]: We should try to avoid adding console.log statements in production builds (it's fine for debugging)

**Comment by MilanDonhowe** [comment]
Ok, I just pushed a change to fix some of the reactivity.  When I was testing the branch, the pop-up wouldn't immediately change its layout when the screen height changed.  I adjusted it by referencing the code in this [StackOverflow post](https://stackoverflow.com/a/50490759/8638218).

We should probably try and adjusting the image for the horizontal layout to take up more space (so there isn't some gray dead space above and below the image) and also get some design feedback possibly from other people in the office.

**Comment by solderq35** [comment]
I reduced some of the dead space and also stretched out the image vertically. This changed the image dimensions a bit but I think it looks ok still.

Comparison of new changes:

wide layout:
wide

vertical layout:
vertical

**Comment by solderq35** [comment]
brandon said the change looks good in today's meeting so might merge later.

**Pull request merged**